snap/snapcraft.yaml

```diff
@@ -1,36 +1,56 @@
-name: saber
-version: 0.9.4
-summary: Handwritten notes
-description: A cross-platform libre handwritten notes app, available for Android, iOS, Linux, Windows, and macOS!
+name: saber # you probably want to 'snapcraft register <name>'
+base: core18 # the base snap is the execution environment for this snap
+version: '0.9.4' # just for humans, typically '1.2+git' or '1.3.2'
+summary: A cross-platform libre handwritten notes app # 79 char long summary
+description: |
+  A cross-platform libre handwritten notes app, available for Android, iOS, Linux, Windows, and the web!
+grade: stable # must be 'stable' to release into candidate/stable channels
+confinement: strict # use 'strict' once you have the right plugs and slots
+license: GPL-3.0+
+compression: lzo
+architectures:
+  - build-on: amd64
 
-confinement: strict
-base: core18
-grade: devel
-
-slots:
-  dbus-saber:
-    interface: dbus
+plugs:
+  saber-svc:
     bus: session
+    interface: dbus
     name: com.adilhanney.saber
-
-apps:
-  saber:
-    command: saber
-    extensions: [flutter-stable]
-    plugs:
-      - network
-    slots:
-      - dbus-saber
-
 parts:
   saber:
+    # See 'snapcraft plugins'
+    plugin:  flutter
     source: .
-    plugin: flutter
-    flutter-target: lib/main_snap.dart
```

Comment:
`flutter-target` needs to remain here

Reply:
That's by default. We should use it if we use something else.

https://snapcraft.io/docs/flutter-plugin

Reply:
> > It'll be the best if you transfer ownership of this, or you may also allow me to write in this repo.
> 
> I have no plans currently to maintain the snap version as I don't wish to install snap on my system. I'm also not willing to give out unrestricted write access to the repo. Please make any future changes if needed through a PR.

Snap builds are currently done from my repo and not yours, so, I'll have the control of it, not you. If that's not a problem, then fine.

Reply:
I think you might have replied to the wrong comment here